### ISSUE TITLE
Multiple twitch videos with the same name

### ISSUE THREAD

**Issue by djmattyg007** [MrS0m30n3/youtube-dl-gui#451]
### What is the purpose of your *issue*?

Bug report

---

### Bug report

A couple of days ago I was streaming on twitch, and had to temporarily stop the broadcast halfway through the stream. I didn't change the stream title before restarting the broadcast, which means I have two VODs with the same name. When I try to download these simultaneously with Youtube-DLG, the second link fails almost immediately. This appears to be happening because the destination filenames are identical.

A week ago I also observed a similar situation, but instead of downloading the two VODs simultaneously, I downloaded them one after the other. In that case, the second download overwrote the first download completely.

#### What operating system do you use ?

Windows 7

#### List of actions to perform to reproduce the problem:

  1. Attempt to download two videos from the same site that have the same name simultaneously.
  
#### What is the expected behaviour ?

All downloads should complete successfully. I suggest adding a unique identifier to each filename, or prompting the user to enter the destination filename before the download starts.

#### What happens instead ?

One of the downloads errors out immediately.

**Comment by axcore** [comment]
Click the button in the top-right corner, and select Options

In the new window, find the dropdown box **Filename format**, and select **Title + ID**

**Comment by djmattyg007** [comment]
While I'm sure that would help, it would be nice if the default configuration didn't result in errors that aren't actually explained to the user by the program itself.

**Comment by axcore** [comment]
If you didn't already know, this project is no longer being maintained.

**Comment by djmattyg007** [comment]
No, I wasn't aware of that. It's quite sad as the app still works perfectly fine and is fantastic for non-technical users. My wife uses it on a regular basis.

**Comment by axcore** [comment]
There is a fork [here](https://github.com/axcore/tartube) that is still being maintained, if you ever run into problems.
